Sources/KSCrashRecording/include/KSCrashReportWriter.h

```diff
@@ -204,7 +204,7 @@ typedef struct KSCrashReportWriter
      *
      * @param name The name to give this element.
      *
-     * @param value A pointer to the JSON data.
+     * @param jsonElement A pointer to the JSON data.
```

Comment:
Might be good to add `closeLastContainer` parameter too then :)

Reply:
Ah yeah, good call.